test/unit/TestVoid.hs

```diff
@@ -0,0 +1,19 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+-- | Swarm unit tests
+module TestVoid where
+
+import Test.Tasty
+import Test.Tasty.HUnit
+import Swarm.Language.Types
+import Swarm.Language.Typecheck (isSimpleUType)
+
+testVoid :: TestTree
```

Comment:
This test is fine --- good job noticing that `isSimpleUType` had better return `True` for `void`, I wouldn't have thought of that.  Fortunately, the way `isSimpleUType` is implemented, we get this for free.